tests/unit/type/set.tcl

```diff
@@ -1029,111 +1029,6 @@ foreach type {single multiple single_multiple} {
         r srem $myset {*}$members
     }
 
-    proc verify_rehashing_completed_key {myset table_size keys} {
-        set htstats [r debug HTSTATS-KEY $myset]
-        assert {![string match {*rehashing target*} $htstats]}
-        return {[string match {*table size: $table_size*number of elements: $keys*} $htstats]}
-    }
-
-    test "SRANDMEMBER with a dict containing long chain" {
```

Comment:
I deleted this test because hashset does not have linked list chains the way that dict does, so the aspect this is attempting to test no longer exists.

Reply:
Hmm, I think this should be a hashset UT instead. Assuming that our random sampling doesn't follow probe chains we should be unaffected, but we want to guard against regressions in the future. My UT would make a hashset with one long chain of similar elements, then ensure those elements aren't under or over represented.

Reply:
Sounds good. Can you write that UT in another PR towards the hashset branch?